spec/unit/recipes/default_spec.rb

```diff
@@ -11,3 +11,15 @@
     expect { :chef_run }.to_not raise_error
   end
 end
+
+describe 'default recipe on Ubuntu 14.04' do
+  let(:chef_run) do
+    ChefSpec::ServerRunner.new do |node|
+      node.automatic[:platform_family] = 'rhel'
```

Comment:
Check out the Chefspec page for the proper way to mock out a CentOS node completely. This will technically work, but it should be done the proper way.